proposals/solid-oidc/primer.bs

```diff
@@ -0,0 +1,745 @@
+ - `"iat": 1603389616`: The date the token was issued, in this case October 22, 2020 14:00:16 GMT.
+ - `"exp": 1603393216`: The date this token will expire, in this case October 22, 2020 15:00:16 GMT.
+
+#### 3. Sends request
+
+With the access token and new DPoP token in hand, we're ready to make our request.
+
+```
+GET https://bob.otherpod.example/private/photo_album.ttl
+Headers: {
+  authorization: "DPOP eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJodHRwczovL2FsaWNlLmNvb2xwb2QuZXhhbXBsZS9wcm9maWxlL2NhcmQjbWUiLCJ3ZWJpZCI6Imh0dHBzOi8vYWxpY2UuY29vbHBvZC5leGFtcGxlL3Byb2ZpbGUvY2FyZCNtZSIsImlzcyI6Imh0dHBzOi8vc2VjdXJlYXV0aC5leGFtcGxlIiwiYXVkIjoic29saWQiLCJjbmYiOnsiamt0IjoiOVhtd0s4bVEzSDUtUG56QXQzbEZIeldCV192NVFoWXluZXpiYml0NGtDOCJ9LCJjbGllbnRfaWQiOiJodHRwczovL2RlY2VudHBob3Rvcy5leGFtcGxlL3dlYmlkI3RoaXMiLCJqdGkiOiJmZDBhMTM1My0yNWYzLTRjN2UtOGY4Yi1jMTQyNjRmMWMxMmUiLCJpYXQiOjE2MDMzODUyNjEsImV4cCI6MTYwMzM4NTg2MX0.HIIYiovILPvsdkD3s3xomR1MkA_Ir8Mx_C-eHxbWEnw9Z2rv2rOWbvMPZ--BqH3qVSupgYTZZsTDbZXA8giCLA",
+  dpop: "eyJhbGciOiJFUzI1NiIsInR5cCI6ImRwb3Arand0IiwiandrIjp7Imt0eSI6IkVDIiwia2lkIjoiQ21HVE9Dd3ZKWXhrb0dGOGNxcFpBNTdab2xVdThBcFJQb3MwVlduWk1TNCIsInVzZSI6InNpZyIsImFsZyI6IkVDIiwiY3J2IjoiUC0yNTYiLCJ4IjoiU0FZcmF5VUh4Z1FPQ29YSC1MbHdyOW1iSWJpUHBsLXRQRUpLeE1WWFltcyIsInkiOiJ6eGJQODdPQ3JpeEZpMk9vZjU1QkhsTC1ySHhvWHVuUmttNFBkV3duUzJnIn19.eyJodHUiOiJodHRwczovL2JvYi5vdGhlcnBvZC5leGFtcGxlL3ByaXZhdGUvcGhvdG9fYWxidW0udHRsIiwiaHRtIjoiZ2V0IiwianRpIjoiZmIxMjY0ZGQtZmZmMS00NTA5LWE3YjEtMGZlNThkMDhkM2UxIiwiaWF0IjoxNjAzMzg5NjE2LCJleHAiOjE2MDMzOTMyMTZ9.G8JktoMOadenCYtO4Z_ZI7ACnjKJvT59OyKlQ6WpB1Qq2GoCK6v1ocrpsfELDOKIL5nt5fwWccfvCAA2bMrkjA"
+}
+```
+ - `headers.authorization: "DPOP eyJhbGciOiJFUzI1N..."`: The authorization header contains the ACCESS TOKEN that we received from the OS during the auth phase. Putting "DPOP" followed by a space tells the Resource Server that this is a DPOP bound access token.
+ - `dpop: "eyJhbGciOi..."`: The dpop token we just generated.
+
+#### 4. Checks DPoP and Access Token expirations
+
+The RS checks if the DPoP token and access tokens are still valid by looking at the `exp` claim. If either token does not have an `exp` claim or either token is expired, the OP must reject the request with a 403.
+
+#### 5. Checks the DPoP token url and method
+
+The RS checks the `htu` and `htm` claims of the dpop token. If the `htu` does not match the protocol, origin and path of the request or the `htm` does not correspond the the http method of the request, the OP must reject the request with a 403.
+
+#### 5.1. (Optional) Checks DPoP token unique identifier
+
+The RS can optionally keep track of all DPoP `jti` claims it received. Because a new DPoP token must be generated each time a request is made, no two tokens should have the same `jti`. If the RS receives a DPoP token with a `jti` it has already encoutnered it may reject the request with a 403.
```

Comment:
```suggestion
The RS can optionally keep track of all DPoP `jti` claims it received. Because a new DPoP token must be generated each time a request is made, no two tokens should have the same `jti`. If the RS receives a DPoP token with a `jti` it has already encountered it may reject the request with a 403.
```